SPECS/hunspell/hunspell.spec

```diff
@@ -1,56 +1,59 @@
 %define double_profiling_build 1
-
-Name:      hunspell
-Summary:   A spell checker and morphological analyzer library
-Version:   1.7.0
-Release:   7%{?dist}
-Source0:  https://github.com/hunspell/hunspell/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
-URL:       https://github.com/hunspell/hunspell
-License:   (LGPLv2+ or GPLv2+ or MPLv1.1) and BSD
+Summary:        A spell checker and morphological analyzer library
+Name:           hunspell
+Version:        1.7.0
+Release:        8%{?dist}
+License:        (LGPLv2+ OR GPLv2+ OR MPLv1.1) AND BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+URL:            https://github.com/hunspell/hunspell
+Source0:        https://github.com/hunspell/hunspell/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

Comment:
There is a medium sev CVE against this package.  But it can be bumped to 1.7.1

Reply:
I can do the upgrade and see how that goes.